src/Footer.js

```diff
@@ -0,0 +1,17 @@
+const Footer = () => {
+    return (
+        <div>
+            <footer id="footer" className="footer-1">
+                <div className="footer-copyright">
+                    <div className="container-fluid">
```

Comment:
im not great with bootstrap but i think there is a justify content class that we can do flex start so would just need the <p> tags